automation_use_cases/scan_automation/lib/helpers/scan_automation.py

```diff
@@ -0,0 +1,73 @@
+import logging
+import time
+from .insightappsec import InsightAppSec
+
+
+def create_scan(api_key: str, region: str, settings: dict):
+    url = f"https://{region}.api.insight.rapid7.com/ias/v1"
+    api = InsightAppSec(url=url, api_key=api_key)
+
+    # Step 1 - Grab app name and scan config
+    app_name = settings.get("scan_info").get("app_name")
+    scan_config = settings.get("scan_info").get("config")
+    interval = settings.get("scan_info").get("status_check_interval")
+    if app_name is None or scan_config is None:
+        logging.error("Application name and scan config must be set up in configuration file")
+        return
+    if interval is None:
+        interval = 300
+    elif interval < 60:
+        logging.error("Status check interval must be at least 60 seconds")
+        return
```

Comment:
Maybe instead of returning here, we could print that message and default it to 60 for them.

Reply:
Think I would recommend using whatever value is provided by end user and if not set then default to 60.  May be confusing if you set it to 5 seconds and it just returns.  Calling every second (while not useful) will not cause problems.